src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -73,6 +73,8 @@
     <Compile Include="System\Reflection\Metadata\MetadataReaderProvider.cs" />
     <Compile Include="System\Reflection\Metadata\BlobContentId.cs" />
     <Compile Include="System\Reflection\Metadata\Signatures\PrimitiveSerializationTypeCode.cs" />
+    <Compile Include="System\Reflection\Metadata\TypeSystem\AssemblyDefinition.netstandard2.0.cs" Condition="'$(TargetGroup)' != 'netstandard1.1'" />
```

Comment:
Now that I think if we keep the pattern of the target group file name it should be `netstandard2.1` as we already shipped 2.0.

@danmosemsft @weshaggard any thoughts on this?